cpp/modmesh/buffer/pymod/wrap_ConcreteBuffer.cpp

```diff
@@ -110,9 +110,53 @@ WrapConcreteBuffer::WrapConcreteBuffer(pybind11::module & mod, char const * pyna
         ;
 }
 
+class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapBufferExpander
+    : public WrapBase<WrapBufferExpander, BufferExpander, std::shared_ptr<BufferExpander>>
+{
+
+    friend root_base_type;
+
+    WrapBufferExpander(pybind11::module & mod, char const * pyname, char const * pydoc);
+
+}; /* end class WrapBufferExpander */
+
+WrapBufferExpander::WrapBufferExpander(pybind11::module & mod, char const * pyname, char const * pydoc)
+    : root_base_type(mod, pyname, pydoc)
+{
+    namespace py = pybind11;
+
+    (*this)
+        .def_timed(
+            py::init(
+                [](size_t length)
+                { return wrapped_type::construct(length); }),
+            py::arg("length"))
+        .def_timed(py::init([]()
+                            { return wrapped_type::construct(); }))
+        .def_timed("reserve", &wrapped_type::reserve, py::arg("cap"))
```

Comment:
The argument name "`cap`" is to look like the argument name of `std::vector::reserve` in https://en.cppreference.com/w/cpp/container/vector/reserve